### PR TITLE
Docker image & Fast DDS Suite [15092][15091]

### PR DESCRIPTION
**Pull request by jparisu**
Merge after #30

### CODE REVIEW

File: docs/rst/installation/docker.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+##################################
+DDS Visualizer Plugin Docker image
+##################################
+
+eProsima distributes a Docker image of |ddsvisualizer| with Ubuntu 20.04 as base image.
+This image launches the |plotjuggler| application with the |ddsvisualizer| already installed and sourced.
+
+#.  Download the compressed Docker image in ``.tar`` format from the
+    `eProsima Downloads website <https://www.eprosima.com/index.php/downloads-all>`_.
+    It is strongly recommended to download the image corresponding to the latest version of |ddsrouter|.
```
Comment (antonrv) [inline docs/rst/installation/docker.rst]: ddsrouter? did you mean fastdds?

File: docs/rst/installation/docker.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+#.  Extract the image by executing the following command:
+
+    .. todo::
+
+        check docker tar name
+
+    .. code-block:: bash
+
+        load ubuntu-ddsvisualizer:<version>.tar
```
Comment (antonrv) [inline docs/rst/installation/docker.rst]:
```suggestion
        docker load -i ubuntu-ddsvisualizer:<version>.tar
```

File: docs/rst/installation/docker.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    where ``version`` is the downloaded version of |ddsrouter|.
```
Comment (antonrv) [inline docs/rst/installation/docker.rst]: `fastdds` instead of `ddsrouter`?

File: docs/rst/dds_suite/dds_suite.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+#.  Extract the image by executing the following command:
+
+    .. todo::
+
+        check docker tar name
+
+    .. code-block:: bash
+
+        load ubuntu-fastdds-suite:<version>.tar
```
Comment (antonrv) [inline docs/rst/dds_suite/dds_suite.rst]:
```suggestion
        docker load  -i ubuntu-fastdds-suite:<version>.tar
```

File: docs/rst/installation/docker.rst (hunk 1 above)
Comment (jparisu) [reply docs/rst/installation/docker.rst]: you may be seeing an old commit

**Review by antonrv — APPROVED** [review]
good to me

**Pull request merged**